bugwarrior/config/schema.py

```diff
@@ -151,17 +179,14 @@ class Config(PydanticConfig):
     #: Interactive status.
     interactive: bool
 
-    # added during validation (computed field support will land in pydantic-2)
-    #: Local data storage.
-    data: typing.Optional[BugwarriorData] = None
+    # optional
+    taskrc: TaskrcPath = Field(default_factory=get_default_taskrc)
 
-    @pydantic.v1.root_validator
-    def compute_data(cls, values):
-        values['data'] = BugwarriorData(get_data_path(values['taskrc']))
-        return values
+    @computed_field
+    @property
+    def data(self) -> BugwarriorData:
```

Comment:
Noticed that the data attribute was removed from the docs in the api-change-check. I think this will put it back.

```suggestion
    def data(self) -> BugwarriorData:
        """Local data storage."""
```